sdk/python/feast/type_map.py

```diff
@@ -486,10 +486,12 @@ def pa_column_to_proto_column(
 
 def bq_to_feast_value_type(bq_type_as_str):
     type_map: Dict[ValueType, Union[str, Dict[str, Any]]] = {
-        "DATETIME": ValueType.STRING,  # Unsure if string is right
-        "TIMESTAMP": ValueType.STRING,  # Unsure if string is right
+        "DATETIME": ValueType.STRING,  # Update to ValueType.UNIX_TIMESTAMP once #1520 lands.
+        "TIMESTAMP": ValueType.STRING,  # Update to ValueType.UNIX_TIMESTAMP once #1520 lands.
+        "INTEGER": ValueType.INT32,
         "INT64": ValueType.INT64,
         "STRING": ValueType.STRING,
+        "FLOAT": ValueType.FLOAT,
```

Comment:
it's pretty weird on BigQuery's part that it's giving different types based on whether you query the INFORMATION_SCHEMA table or look at the field_type on schema_field; however, it seems INTEGER and INT64 are equivalent, and same for FLOAT and FLOAT64 - see https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#TableFieldSchema.FIELDS.type. 
So I think that here you can just have "INTEGER" map to the int64 value type and "FLOAT" map to the double value type, and this should also fix the mapping to different types in the test below